Orange/canvas/application/canvasmain.py

```diff
@@ -1057,6 +1061,23 @@ def set_new_scheme(self, new_scheme):
 
         old_scheme.deleteLater()
 
+    def ask_clear_report(self):
+        from Orange.canvas.report.owreport import OWReport
```

Comment:
I'm not sure what your policy on imports is, but in the contributing.txt it says "Whenever PEP-8 is undefined, adhere to Google Python Style Guide." That would mean all imports must be on the top of the file.